File: lib/app/Models/UpdateStockItemDTO.php
<?php
/**
 * UpdateStockItemDTO
 */
namespace app\Models;

/**
 * UpdateStockItemDTO
 * @description Информация об остатках товара.
 */
class UpdateStockItemDTO {

    /** @var int $count Количество доступного товара.*/
    public $count = 0;

    /** @var \DateTime $updatedAt Дата и время последнего обновления информации об остатках. &lt;br&gt;&lt;br&gt; Если вы не передали параметр &#x60;updatedAt&#x60;, используется текущее время. &lt;br&gt;&lt;br&gt; Формат даты и времени: ISO 8601 со смещением относительно UTC. Например, &#x60;2017-11-21T00:42:42+03:00&#x60;.*/
    public $updatedAt;

}

File: src/main/scala/org/openapitools/models/UpdateStockItemDTO.scala
package org.openapitools.models

import io.circe._
import io.finch.circe._
import io.circe.generic.semiauto._
import io.circe.java8.time._
import ympa_scala_finch_server._
import java.time.ZonedDateTime

/**
 * Информация об остатках товара. 
 * @param count Количество доступного товара. 
 * @param updatedAt Дата и время последнего обновления информации об остатках. <br><br> Если вы не передали параметр `updatedAt`, используется текущее время. <br><br> Формат даты и времени: ISO 8601 со смещением относительно UTC. Например, `2017-11-21T00:42:42+03:00`. 
 */
case class UpdateStockItemDTO(count: Long,
                updatedAt: Option[ZonedDateTime]
                )

object UpdateStockItemDTO {
    /**
     * Creates the codec for converting UpdateStockItemDTO from and to JSON.
     */
    implicit val decoder: Decoder[UpdateStockItemDTO] = deriveDecoder
    implicit val encoder: ObjectEncoder[UpdateStockItemDTO] = deriveEncoder
}
